### PR TITLE
Change license to MIT

### LINKED ISSUE #152: Change license
We have decided to change the license to MIT.

### PR DESCRIPTION
**Pull request by jwallwork23**
We have decided to change Pyroteus' license to MIT.

I also included an ``AUTHORS`` file for everyone that has opened or reviewed Pyroteus pull requests. Please could you all take a look at this to check you are happy with how your name is presented. Feel free to change your affiliation if something other than Imperial is more appropriate. Thanks!

Closes #152.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Pyroteus has received contributions from:
+
+Joseph G. Wallwork, Met Office, UK, <joe.wallwork@outlook.com>
+Stephan C. Kramer, Imperial College London, UK
+Akhil Mohan, Imperial College London, UK
+Eleda Johnson, Imperial College London, UK
+Davor Dundovic, Imperial College London, UK
```
Comment (ddundo) [inline AUTHORS:7]: Mine should probably say University of Oslo, Norway since I'm not officially affiliated with ICL :)

**Review by ddundo — CHANGES_REQUESTED** [review]

File: AUTHORS (hunk 1 above)
Comment (jwallwork23) [reply AUTHORS]: Done. See c1895291f16bf8ac447f85a3ccb2154dd23497ef.

**Review by ddundo — APPROVED** [review]

File: AUTHORS (hunk 1 above)
Comment (ddundo) [reply AUTHORS]: Thanks Joe!

**Review by acse-sl420 — APPROVED** [review]

**Review by stephankramer — APPROVED** [review]
Perfect, thanks Joe

**Pull request merged**